README.md

```diff
@@ -1,6 +1,71 @@
 [![Build Status](https://travis-ci.org/openmrs/openmrs-module-calculation.svg?branch=master)](https://travis-ci.org/openmrs/openmrs-module-calculation)
 
-openmrs-module-calculation
-==========================
+# OpenMRS Calculation Module
 
-A framework for calculating clinical rules in OpenMRS and making them available for other modules
\ No newline at end of file
+The Calculation module provides a framework for OpenMRS developers to author from simple to complex calculations to produce a particular piece of data for a single patient or a cohort of patients. It is a high level abstraction of rules/calculations, simple in that it provides a minimal number of useful concrete implementations of its core components that can be used out of the box but providing a much more flexible tool for authoring patient centered calculations.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the Calculation Module on a live system.
+
+### Prerequisites
+
+Follow instructions on OpenMRS web site to download and install the OpenMRS SDK, found at https://wiki.openmrs.org/display/docs/OpenMRS+SDK+Step+By+Step+Tutorials
+
+### Installing
+Download the Calculation Module. On the OpenMRS website, you can find a compiled version of the module at https://addons.openmrs.org/#/show/org.openmrs.module.calculation. Upload this file into your system. See instructions below if you need instructions on how to upload the file into your system.
+
+You may also download the source code from the Calculation Module on github. This is useful for making changes on the module. Follow the following instructions to do this. 
+
+```
+git clone https://github.com/openmrs/openmrs-module-calculation.git
+cd openmrs-module-calculation
+mvn clean install
+```
+
+Find the .omod file in openmrs-module-calculation/omod/target. Upload this file into your system.
+
+To upload the file to your system, find your OpenMRS modules folder, which should be labeled
+
+```
+/openmrs/modules/
+```
+
+Drag your .omod file into this folder, if it does not already contain one.
+
+From here, run your OpenMRS server and navigate to 
+
+```
+http://localhost:8080/openmrs/initialsetup/
```

Comment:
How about those whose instance name is not openmrs and are not running on port 8080?